tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -679,10 +679,10 @@ ifneq ($(findstring $(EXPLICITLY_SPECIFIED_TEST),$(MICROLITE_TEST_SRCS)),)
   MICROLITE_TEST_SRCS := $(filter-out $(EXPLICITLY_SPECIFIED_TEST), $(MICROLITE_TEST_SRCS))
   EXPLICITLY_SPECIFIED_TEST_SRCS := \
   $(EXPLICITLY_SPECIFIED_TEST) \
-  tensorflow/lite/micro/benchmarks/keyword_scrambled_model_data.cc \
+  $(GENERATED_SRCS_DIR)tensorflow/lite/micro/models/keyword_scrambled_model_data.cc \
```

Comment:
It's odd to me that this was in the Makefile to start with. What happens if we remove it?

Reply:
The memory_arena_threshold_test fails to compile with the Makefile without this.

Reply:
this should depend on the tflite file, right? Else the build will fail if we no longer do the tflite to model_data generation as part of the keyword_benchmark.